tests/python/unittest/test_sparse_operator.py

```diff
@@ -171,9 +171,9 @@ def check_csr_slice(shape, slice_input):
 
 
 def test_sparse_retain():
```

Comment:
should `test_sparse_retain ` be added to test_operator_gpu since cast_storage is completed?